tutorons/python/detect.py

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import logging
+import ast
+
+from tutorons.common.extractor import Region
+from tutorons.python.explain import explanations
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+def filter_non_ascii(c):
+    if ord(c) > 127:
+        return ' '
+    return c
+
+
+def findOffset(text_list, call):
+    """Takes in a text as a list and function call and finds the start of call within text"""
+    offset = 0
+    if call.lineno > 1:
+        for l in xrange(call.lineno - 1):
```

Comment:
Might be able to simplify this by removing the `if` statement above and writing your for loop like:
    for l in range(1, call.lineno):

That would skip index 1 and go up until `call.lineno - 1`